snmp/ups-apcups.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+################################################################
+# copy this script to somewhere like /opt and make chmod +x it #
+# edit your snmpd.conf and include                             #
+# extend ups-apcups /opt/ups-apcups.sh                         #
```

Comment:
That is why I used that in my documentation entries, so I would prefer /etc/snmp